ee/clickhouse/queries/breakdown_props.py

```diff
@@ -78,7 +78,7 @@ def get_breakdown_event_prop_values(
         filter.properties, team_id, table_name="e", filter_test_accounts=filter.filter_test_accounts,
     )
 
-    entity_params, entity_format_params = populate_entity_params(entity)
+    entity_params, entity_format_params = populate_entity_params(entity, team_id, with_prop_clauses=True)
```

Comment:
The only other callsite here is ee/clickhouse/queries/trends/trend_event_query.py. Should that include property filters as well?

Reply:
Nope, that callsite doesn't need prop filters because it inserts them elsewhere. The breakdown values query is more lean though and it does need those filters right here.

Reply:
We have two callsites for a function with a boolean parameter - in essence we're calling two different functions. Does unifying the two even make sense or perhaps we should refactor the trends query to use the filters injected from there??

Reply:
After playing around with this, I think the current state is fine – there isn't duplication of filters, and queries don't end up malformed (not due to this). When trying out approaches with params injected differently, correctness was a bit weird to nail, so while small of a refactor would be nice, I'll table it until we see there are material improvements at stake.